build.gradle

```diff
@@ -1,8 +1,6 @@
 plugins {
     id 'groovy'
-    id 'java-gradle-plugin'
-    id 'com.gradle.plugin-publish' version '0.11.0'
-    id 'com.palantir.git-version' version '0.11.0'
```

Comment:
Plugin `com.palantir.git-version' is needed for automatic version resolving based on tag name and commit distance. 

The original idea here was to set tag for minor/major version and then just use commit distance for patch version and release on each commit to master. Nowadays I find it bit complicated so we can get rid of it. But this could be done as follow up change and kept here as it is. Otherwise pipeline for publishing will fail.

Reply:
Reverted